src/components/appHeader/AppHeader.js

```diff
@@ -22,11 +22,11 @@ const AppHeader = () => {
                             to="/"
                         >
                             Characters
-                        </NavLink>
+                        </RouterLink>
                     </li>
                     /
                     <li>
-                        <NavLink
+                        <RouterLink
                             activeStyle={{
```

Comment:
can we craete active_style variable and use it in all places ?

Reply:
I have created active style for navlink just in this component because it use only here